openedx/features/edly/middleware.py

```diff
@@ -27,6 +31,24 @@ def process_request(self, request):
         if request.user.is_superuser or request.user.is_staff:
             return
 
+        edly_sub_org = getattr(request.site, 'edly_sub_org_for_lms', None)
+        if edly_sub_org:
+            if not is_edly_sub_org_active(edly_sub_org):
+                logger.exception('EdlySubOrganization for site %s is disabled. ', request.site)
+                marketing_url = marketing_link('ROOT')
+
+                if marketing_url:
+                    return HttpResponseRedirect(marketing_url)
+                else:
```

Comment:
Does this condition get true in any case?

Reply:
The marketing_url gets the value from the marketing_link('ROOT') function. 
It will either return the marketing URL if found in the site configurations or else return the default value of '#' if the URL is not found in configurations or settings. 

The condition has been updated to :
                                    if marketing_url != '#':